GpsGarmin:
  address: "mrs_uav_state_estimation/GpsGarmin"
